src/main/java/com/webank/weid/constant/ErrorCode.java

```diff
@@ -280,13 +280,11 @@
     CREDENTIAL_USE_VERIFY_FUNCTION_ERROR(100439,
         "presentation from pdf transportation, please use verifyPresentationFromPDF function"),
 
-
     /**
-     * lite credential does not support selective disclosure.
+     * the credential does not support selective disclosure.
```

Comment:
the error code shows that the credential passed in the function is not supported by this function.